content/en/getting_started/integrations/google_cloud.md

```diff
@@ -0,0 +1,351 @@
+      tag: 'Blog'
+---
+
+## Overview
+
+Integrating your Google Cloud projects with Datadog allows you to monitor your Google Cloud services within the Datadog platform.
+This guide provides an overview of how to get started with the Datadog Google Cloud integration.
+
+## Prerequisites
+1) Create a [Datadog account][1]
+2) Set up a [Service Account][2] in any of your Google Cloud projects
+3) Review these Google Cloud Prerequisites:
+
+{{% site-region region="us,us3,us5,eu,ap1" %}}
+- If your organization restricts identities by domain, you must add Datadog's customer identity `C0147pk0i` as an allowed value in your policy.
+{{% /site-region %}}
+- Datadog does not support [scoping projects][42] that pull in metrics from multiple other projects.
+- The Google Cloud integration requires the below APIs to be enabled **for each of the projects** you want to monitor:
+
+[Cloud Monitoring API][7] 
+: Allows Datadog to query your Google Cloud metric data.
+
+[Compute Engine API][8] 
+: Allows Datadog to discover compute instance data.
+
+[Cloud Asset API][9]
+: Allows Datadog to request Google Cloud resources and link relevant labels to metrics as tags.
+
+[Cloud Resource Manager API][10] 
+: Allows Datadog to append metrics with the correct resources and tags.
+
+[IAM API][11]
+: Allows Datadog to authenticate with Google Cloud.
+
+[Google Cloud Billing API][12] 
+: Enables accurate billing for API calls.
+
+<div class="alert alert-info">You can confirm if these APIs are enabled by heading to <a href="https://console.cloud.google.com/apis/dashboard"><b>Enabled APIs & Services</a></b> under each project in your Google Cloud workspace.</div>
+
+## Setup 
+
+For the most comprehensive level of monitoring in your Google Cloud environment, integrate Datadog with [Google Cloud][4] using [service account impersonation][5] at the organization or folder level.
+
+After reviewing the above [Prerequisites][3], follow these step-by-step instructions to set up the Google Cloud integration:
+
+{{% collapse-content title="1. Create a Google Cloud service account in the default project" level="h5" %}}
+1. Open your [Google Cloud console][53].
+2. Navigate to **IAM & Admin** > **Service Accounts**.
+3. Click **Create service account** at the top.
+4. Give the service account a unique name.
+5. Click **Done** to complete creating the service account.
+{{% /collapse-content %}} 
+
+{{% collapse-content title="2. Add the service account at the organization or folder level" level="h5" %}}
+1. In the Google Cloud console, go to the **IAM** page.
+2. Select a folder or organization.
+3. To grant a role to a principal that does not already have other roles on the resource, click **Grant Access**, then enter the email of the service account you created earlier.
+4. Enter the service account's email address.
+5. Assign the following roles:
+- Compute Viewer
+- Monitoring Viewer
+- Cloud Asset Viewer
+- Browser
```

Comment:
Yes true but why?

Reply:
Are you suggesting adding a justification for each role assignment for the customer, would that be better served in our general docs instead of this getting started guide that's meant to be a quick onboarding?